controllers/articles.controllers.js

```diff
@@ -2,80 +2,47 @@ const { selectArticleById, selectArticles } = require("../models/articles.models
 const { selectArticleComments } = require("../models/comments.models");
 
 
+  
 exports.getArticleById = (request, response, next) => {
     const { article_id } = request.params;
-    if (isNaN(article_id)) {
-      return next({ status: 400, message: "Invalid ID" });
-    }
+  
     selectArticleById(article_id)
       .then((article) => {
-        if (!article) {
-          return next({ status: 404, message: "Article not found" });
-        }
         response.status(200).send({ article });
       })
       .catch(next);
   };
 
-
 exports.getArticles = (request, response, next) => {
-    const { sort_by, order, article_id } = request.query;
-
+    const { sort_by, order, topic, article_id } = request.query;
+  
     if (article_id) {
```

Comment:
It's not clear to me why you have this here. Did this controller used to be for /api/articles/:article_id? I've checked you app.js and you have a separate controller to handle that endpoint now. I'd suggest removing this chunk.